pyproject.toml

```diff
@@ -2,7 +2,6 @@
 # Must be kept in sync with `setup_requirements` in `setup.py`
 requires = [
     "setuptools>=40.8.0",
-    "wheel",
```

Comment:
Not withstanding the comments in the old PR, as I understand PEP 517/518 this value needs to stay in the requires for the build-backend (see: https://www.python.org/dev/peps/pep-0518/).

The de-synchronization is an unfortunate consequence, but why should Python packaging be consistent now 😄